samples/scenarios/jdbc_kerberos/manifest.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version='1.0' encoding='utf-8' ?>
+
+<connector-plugin class='postgres_jdbc_kerberos' superclass='jdbc' plugin-version='0.0.0' name='PostgreSQL JDBC with Kerberos' version='18.1' min-version-tableau='2020.3'>
```

Comment:
```suggestion
<connector-plugin class='postgres_jdbc_kerberos' superclass='jdbc' plugin-version='0.0.0' name='PostgreSQL JDBC with Kerberos' version='18.1' min-version-tableau='2020.4'>
```
Double check that this can run in 2020.3, if not commit above change

Reply:
Fixed. Confirmed with dsoni about this.

Reply:
This was tested when near branch was at 2020.4 so it is safe to change min version to 2020.4.